daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/VersionTimeline.scala

```diff
@@ -54,6 +54,7 @@ private[lf] object VersionTimeline {
       That(LanguageVersion(LMV.V1, "5")),
       This(That(TransactionVersion("8"))),
       This(This(ValueVersion("5"))),
+      That(LanguageVersion(LMV.V1, "6")),
```

Comment:
If value version 5 wasn't released with the SDK yet, language 1.6 should be Both'd with value 5. Otherwise good like this.

Reply:
Fixed